src/track/track.h

```diff
@@ -255,8 +252,9 @@ class Track : public QObject {
     void removeCue(const CuePointer& pCue);
     void removeCuesOfType(mixxx::CueType);
     QList<CuePointer> getCuePoints() const;
+
     void setCuePoints(const QList<CuePointer>& cuePoints);
-    void importCuePoints(const QList<mixxx::CueInfo>& cueInfos);
+    void importCues(const QList<mixxx::CueInfo>& cueInfos);
```

Comment:
Some functions are using `CuePoints`, some are using `Cues`. Should we try to settle on a single name?

Reply:
I have chosen the term "Cue" for everything dealing with plain `CueInfo` data. The traditional term "CuePoint" is used for the class `Cue` whose objects are passed as `CuePointer` and are living objects, not just pure data classes. We should not mix this up.

Suggestions how to deal with the situation are welcome! I am well aware that this two-fold design is confusing and far from ideal. But instead of hiding the differences we should make them explicit. It indicates that we are mixing up different abstraction levels here which is not good. But we cannot solve this now.

Reply:
How about naming it ` importCueInfos`?

Reply:
The generic "Info" suffix was just added for disambiguation to those data classes. I wanted to prevent spreading it everywhere, because it doesn't convey any information.

Reply:
It's quite ironic that "Info" doesn't convey any information :laughing:

Reply:
...like naming things that *do stuff* "Manager", same story.